examples/Terminal_Velocity.ipynb

```diff
@@ -0,0 +1,295 @@
+    "\n",
+    "---\n",
+    "\n",
+    "This notebook discusses the <a href=\"http://en.wikipedia.org/wiki/Drag_(physics)\">drag forces</a> exerted on a body when traveling through air. It's also done in [Julia](julialang.org) so that I can better understand the potencial of the language.\n",
+    "\n",
+    "![](http://upload.wikimedia.org/wikipedia/en/8/89/Parachuters_in_hybrid_formation.jpg)  \n",
+    "\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A falling body is subject to two forces: a downward force, $\\vec{F_g}$, due to gravity, and an upward force $\\vec{F_a}$, due to air resistance. Let's consider a body with mass $m$. Assuming an upward oriented YY axis, the following applies:\n",
+    "\n",
+    "$$ F_g = - g \\cdot m $$\n",
+    "\n",
+    "$$ F_a = - sgn(v) \\cdot {1 \\over 2} \\rho C_D A v^2 $$\n",
+    "\n",
+    "In these formulas, $m$ is the mass of the body, $\\rho$ is the density of air ($\\approx 1.2 Kg/m^3$), $g$ is the accelaration of gravity ($\\approx 9.8 m/s^2$), $v$ is the velocity of the body, $C_D$ is the drag coeficient of the body and $A$ is its cross-section area. $sgn(x)$ is the sign function that given a number returns +1 for positives and -1 for negatives. In the formula it makes the force always contrary to the movement of the body.\n",
+    "\n",
+    "Thus, the resulting force $\\vec{F}$ experienced by the body is:\n",
+    "\n",
+    "$$ F = - g \\cdot m - sgn(v) \\cdot {1 \\over 2} \\rho C_D A v^2 $$\n",
+    "\n",
+    "Given that force is directly related to accelaration, and accelaration to velocity:\n",
+    "\n",
+    "$$ v = \\frac{dy}{dt} $$\n",
+    "\n",
+    "$$ F = m \\cdot \\frac{dv}{dt} $$\n",
+    "\n",
+    "substituting, we get this differencial equation:\n",
+    "\n",
+    "$$ m \\cdot \\frac{dv}{dt} = - g \\cdot m - sgn(v) \\cdot {1 \\over 2} \\rho C_D A v^2 $$\n",
+    "\n",
+    "In fact, this can be expressed as a system of differential equations in the form ${\\bf f}' = {\\bf g}({\\bf f}, t)$:\n",
+    "\n",
+    "$$ \\begin{cases} \\frac{dy}{dt} = v \\\\\\\\ \\frac{dv}{dt} = -g - sgn(v) \\cdot {1 \\over 2} {\\rho \\over m} C_D A v^2 \\end{cases} $$\n",
+    "\n",
+    "---\n",
+    "\n",
+    "So, let's simulate this with Julia.\n",
+    "\n",
+    "Let's start by importing some basic libraries:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "using PyPlot\n",
+    "using ODE;"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We now define the initial conditions and constants of the problem."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# Constants \n",
+    "g  = 9.8                # Accelaration of gravity\n",
```

Comment:
It is bad style and bad for performance to use non-const globals.  Instead use `const g =9.8` etc.